File: modelo/formula.cs
using System.Data;
using System;
public class formula : oObjeto
{

    public string Script { get; set; }
    public string Titulo { get; set; }
    public int Estado { get; set; }
    public formula() { }
    public formula(DataRow dr)
    {
        this.Id = Convert.ToInt32(dr["id"]);
        this.Script = dr["script"].ToString();
        this.Estado = Convert.ToInt32(dr["estado"]);
        this.Titulo = dr["titulo"].ToString();

    }
}

File: ClientApp/src/app/clases/formula.ts
export class formula
{
    id: number; 
    estado: number;
    script: string; 
    titulo:string;

    constructor(id:number,script:string,estado:number,titulo:string)
    {
        this.id = id;
        this.estado = estado;
        this.script =  script;
        this.titulo=titulo;
 
    }
}